graphql-dgs-example-java/src/main/resources/application.yml

```diff
@@ -15,5 +15,7 @@ logging:
 
 dgs:
   graphql:
+    introspection:
```

Comment:
Did you leave this by purpose in the config file? Same for the config file in the other example project

Reply:
@paulbakker I did leave it on purpose to show how it would look like but I am noticing we don't do that for most config options, removed from both application.yml